### PR TITLE
Canada visa intake

### LINKED ISSUE #67: Visa eligibility checks - Canada (first section)
As a destination team member,
I would like to complete a candidate's visa eligibility check on the Talent Catalog,
So that I can easily view the data (intake data, job data) and make the necessary visa check assessments in the one place (the TC).

### PR DESCRIPTION
**Pull request by cazcam34**
Not finished yet, still working on this branch. See issue for checkbox of what is still to do. I am hoping to get the first sectio of the visa intake completed this week, so I can get feedback in staging. Then I will start working on the second section (job specific)

### CODE REVIEW

**Comment by cazcam34** [comment]
First section of visa check (visa specific - not the job specific part) for Canada is ready for merge and once merged I'll get feedback from the Canada team. 

Please note: When I merged staging into my branch it's taken my layout of the import section of the files changed, these were then include in my post merge commit. The imports just have a change in spacing, but it looks like i've changed more files than I did. Be good to figure out how to avoid these changes in file layout by having consistent settings between us.

File: ui/admin-portal/src/app/components/candidates/visa/pathway-assessment/pathway-assessment.component.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div *ngIf="error">
+  {{error}}
+</div>
+
+<form [formGroup]="form">
+  <div class="mb-3">
+    <!-- Question for reviewer - could I show different labels for the same field depending on the country?
```
Comment (camerojo) [inline ui/admin-portal/src/app/components/candidates/visa/pathway-assessment/pathway-assessment.component.html]: Remove this comment after acceptance.

File: ui/admin-portal/src/app/components/candidates/visa/pathway-assessment/pathway-assessment.component.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    e.g. if visa country == Canada the label is Meets EMPP Eligibility?
+     However if we use this field for other countries, we could show a different label that's relevant to their pathways?
+     Not sure if this is bad practise -->
+    <label class="form-label" for="visaPathwayAssessment">Meets pathway eligibility? (eg. EMPP for Canada)</label>
```
Comment (camerojo) [inline ui/admin-portal/src/app/components/candidates/visa/pathway-assessment/pathway-assessment.component.html:7]: Probably will also need a notes field (in next release)

**Review by camerojo — COMMENTED** [review]
You said you wanted to do some other stuff before resubmitting for review

**Comment by cazcam34** [comment]
Ready for review

**Review by camerojo — APPROVED** [review]

**Pull request merged**